endpoints-service-config/src/main/java/com/google/api/config/ServiceConfigSupplier.java

```diff
@@ -49,6 +50,25 @@
 public final class ServiceConfigSupplier implements Supplier<Service> {
   private static final String SERVICE_NAME_KEY = "ENDPOINTS_SERVICE_NAME";
   private static final String SERVICE_VERSION_KEY = "ENDPOINTS_SERVICE_VERSION";
+  private static final ImmutableMap<Integer, String> ERROR_CODE_DETAILS =
+      ImmutableMap.<Integer, String>builder()
+          .put(
+              403 /* forbidden */,
+              "This may occur if the App Engine service account has been deleted or does not " +
+              "have correct permissions. Visit https://goo.gl/UEKik4 and verify that the " +
```

Comment:
is it really best practice to use url shorteners in these?

Reply:
I don't see the harm, but if you think it's necessary, I can switch them to the full URL if you prefer. I know that Guava uses short URLs everywhere in their code comments. I think it looks cleaner this way.

Reply:
ok, sounds fine as it is